src/main/scala/symsim/examples/concrete/mountaincar/Mountaincar.scala

```diff
@@ -0,0 +1,110 @@
+package symsim
+package examples.concrete.mountaincar
+
+import symsim.concrete.Randomized
+
+
+object MountainCar
+  extends Agent[CarState, CarFiniteState, CarAction, CarReward, Randomized] {
+    def roundAt(p: Int)(n: Double): Double = { val s = math pow (10, p); (math round n * s) / s }
```

Comment:
My suggestion:
```scala
def roundAt(p: Int)(n: Double): Double = { 
  val s = math.pow (10, p)
  math.round (n * s) / s 
}
```
I am not sure if the parentheses bind correctly with the multiplication in your code (I can't remember, but I am afraid that they don't). I would prefer to add parentheses around argument just in case. I also think that the more classic syntax with dot and parentheses is easier to read to others later.  I like putting one computation per line, also for readability. 

BTW. shouldn't the division be by `n`?

Reply:
@MahsaVarsh when you check the above comment (and possibly address it), you can merge the pull request (or you can tell me to do it).

Reply:
I added the changes to roundAt. Dividing by s should be correct since n is the number that we are rounding.